students/alinafe/codingBat/caught_speeding.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+"""
+You are driving a little too fast, and a police officer stops you. Write code to compute the result, encoded as an int value: 0=no ticket, 1=small ticket, 2=big ticket. If speed is 60 or less, the result is 0. If speed is between 61 and 80 inclusive, the result is 1. If speed is 81 or more, the result is 2. Unless it is your birthday -- on that day, your speed can be 5 higher in all cases.
+"""
+
+def caught_speeding(speed, is_birthday):
+  if is_birthday:
+    speed -= 5
+     
+  if speed <= 60:
+      return 0
+  if 60 < speed <= 80:
+    return 1
+  return 2
+
+if __name__ == "__main__":
+    # some tests
+    caught_speeding(60, False)	is 0
```

Comment:
You don't want to use "is" here -- you want ==.

"is" tests if it is the same object. As it happens, cPython re-uses small integers, so one zero actually IS the same object as another zero -- but that's an accident of implementation.

I'm going to talk about that in class this week.